website/docs/docs/deploy/deployment-tools.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Deploy with other tools"
+id: "deployment-tools"
+sidebar: "Deploy with other tools"
+---
+
+Discover effortless ways to schedule and run your dbt jobs with the help of powerful tools like [dbt Cloud](/docs/deploy/dbt-cloud-job), Airflow, Prefect, Dagster, automation server, Cron, or Azure Data Factory (ADF). 
```

Comment:
Not sure if this captures it:
```suggestion
Discover other ways besides [dbt Cloud](/docs/deploy/dbt-cloud-job) to schedule and run your dbt jobs with the help of powerful tools like Airflow, Prefect, Dagster, automation server, Cron, or Azure Data Factory (ADF). 
```

Reply:
I've tweaked this slightly because I'd like people to know they can use the tools WITH dbt Cloud (trigger cloud jobs), as well as use the dbt Cloud scheduler.